draft-group-tls-external-psk-guidance.md

```diff
@@ -178,24 +178,84 @@ authentication.)
 
 ## Security Assumptions
 
-TODO(jonathan): writeme
-
-- Shared session keys:
-- Session key secrecy:
-- Peer authentication:
-- Session key uniqueness:
-- Downgrade protection:
-- Forward secrecy:
-- KCI protection:
-- Endpoint identity protection:
+As discussed in {{use-cases}}, there are use cases where multiple clients or multiple servers share a PSK. In such
+cases, TLS only authenticates the entire group. Not only can a compromised group member impersonate another group
+member, but a malicious non-member can reroute handshakes between honest group members to connect them in unintended
+ways.
+
+The TLS external PSK authentication mechanism implicitly assumes that each PSK is known to exactly one client and one
+server, and that these never switch roles.  If this assumption is violated, then TLS 1.3 offers virtually no security by
+default.
+
+TLS 1.3 tries to maintain 8 security properties in the handshake.  A naïve sharing of PSKs, even assuming only honest
+but curious participants know the key, can violate all these properties, bar one.  The protection of endpoint identities
+property holds vacuously, because the client and server do not use certificates in PSK mode.
```

Comment:
```suggestion
holds vacuously, because the client and server do not use certificates in PSK mode. (This is not true with use of the "tls_cert_with_extern_psk" extension {{?I-D.ietf-tls-tls13-cert-with-extern-psk}}.)
```